### PR TITLE
Add docstring to ImageManager.__init__ and fix docstring for ImageManager.archive_image

### PR DESCRIPTION
**Pull request by iamjpotts**
##### SUMMARY

Change originally in #500 but pulled this out as a standalone change because that build is failing and to see if this build would fail also.

This change can stand alone without #500.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

`docker_image`

##### ADDITIONAL INFORMATION

No changes to module behavior or output.

### CODE REVIEW

**Comment by iamjpotts** [comment]
@felixfontein 

I think there's an environmental or configuration problem with the CI.

This change shouldn't fail if HEAD on main succeeded.

This change failed CI due to a temporary registry not starting.

File: plugins/modules/docker_image.py [hunk 1]
```diff
@@ -528,12 +536,14 @@ def absent(self):
             self.results['image']['state'] = 'Deleted'
 
     def archive_image(self, name, tag):
-        '''
+        """
```
Comment (felixfontein) [inline plugins/modules/docker_image.py:539]: Please change this back to `'''`, and change the other docstring to `'''` as well.

Comment (iamjpotts) [reply plugins/modules/docker_image.py]: Done.

**Comment by iamjpotts** [comment]
Same two CI builds failed. Is #510 a potential fix for this?

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@iamjpotts thanks for improving the docstrings!

**Pull request merged**